arb_reformatter.py

```diff
@@ -18,6 +18,11 @@ def write_to_file(file_path, text):
     with open(file_path, 'w', encoding='utf-8') as file:
         file.write(text)
 
+# Translation table for unicode characters we want to keep readable.
```

Comment:
Isn't it the opposite?

```suggestion
# Translation table for unicode characters we want to keep in escaped form.
```

Reply:
yeah that's what I meant, I will use your suggestion 🥇 thanks!

Reply:
Fixed!